images/openshift-acme-controller/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-release:golang-1.13 as builder
+FROM quay.io/app-sre/openshift/origin-release:golang-1.13 as builder
```

Comment:
This doesn't exist

You may want to try with registry.access.redhat.com/ubi8/go-toolset:1.13.15